manifests/addons/dashboards/istio-mesh-dashboard.json

```diff
@@ -421,7 +421,7 @@
       "tableColumn": "",
       "targets": [
         {
-          "expr": "avg(galley_istio_networking_virtualservices)",
+          "expr": "pilot_k8s_cfg_events{type=\"VirtualService\"}",
```

Comment:
This is events. Before it was "current number of virtual services". Now we will have 3 series for each instance of istiod (add/update/delete) which will not work ("Only queries that return single series/table is supported")

Reply:
I tried `avg(pilot_k8s_cfg_events{type="VirtualService", event="add"}) - (avg(pilot_k8s_cfg_events{type="VirtualService", event="delete"})` and it seems to almost work but we need to handle delete being empty. I am also not sure it will actually be accurate - it might be but seems potentially flaky

Reply:
what about something like:

```
avg(pilot_k8s_cfg_events{type="VirtualService", event="add"}) - (avg(pilot_k8s_cfg_events{type="VirtualService", event="delete"}) or up * 0)
```

to cover missing timeseries.

there is definitely potential for flakes. we could use `max` instead of `avg` to account for istiods that only see some of the events?